code/modules/station_goals/dna_vault.dm

```diff
@@ -219,5 +219,6 @@
 		if(VAULT_QUICK)
 			to_chat(H, span_notice("Your arms move as fast as lightning."))
 			H.next_move_modifier = 0.5
+	ADD_TRAIT(H, TRAIT_USED_DNA_VAULT, DNA_VAULT_TRAIT)
```

Comment:
update the 6 above it with the define too please.

Reply:
This is given regardless of what you take from the vault